examples/mnist/requirements.txt

```diff
@@ -1,2 +1,2 @@
 pillow==5.4.1
-tensor2tensor==1.10.0
+tensor2tensor==1.12.0
```

Comment:
Their latest [release](https://github.com/tensorflow/tensor2tensor/releases) seems to be `1.13.4`, can we use that?